docs/api/autoplay.mdx

```diff
@@ -25,9 +25,35 @@ For accessibility, the carousel will pause when the user is interacting with it.
 #### Code
 
 ```tsx
-<Carousel autoplay={true} autoplayInterval={1000}  wrapMode="wrap">
+<Carousel autoplay={true} autoplayInterval={1000} wrapMode="wrap">
   <img src="pexels-01.jpg" />
   <img src="pexels-02.jpg" />
   <img src="pexels-03.jpg" />
 </Carousel>
 ```
+
+## Initial page
+
+The carousel can start on any index within bounds of its length. Anything out of bounds will default back to `0` for its index. This list is `0` indexed.
+
+| Prop Name     | Type   | Default Value |
+| :------------ | :----- | :------------ |
+| `initialPage` | number | `0`           |
+
+### Example
+
+<Carousel initialPage={1}>
```

Comment:
The carousel in the docs is the actual nuka version and not what I have locally. Is there a way to get the local changes on the docs?

Reply:
I was hitting a bug that was always starting the carousel at index 0 on the docs site so I thought it wasnt building correctly. This is fixed in [4b15005](https://github.com/FormidableLabs/nuka-carousel/pull/1090/commits/4b150059a2c9993ab3042545d4507229c56e6ba3)